simapp/README.md

```diff
@@ -12,22 +12,24 @@ If you want to spin up a quick testnet with your friends, you can follow these s
 Unless otherwise noted, every step must be done by everyone who wants to participate
 in this testnet.
 
-1. `$ make build`. This will build the `simd` binary and install it in your Cosmos SDK repo, 
-    inside a new `build` directory. The following instructions are run from inside 
-    that directory. 
+1. From the root directory of the Cosmos SDK repository, run `$ make build`. This will build the
+    `simd` binary inside a new `build` directory. The following instructions are run from inside
+    the `build` directory.
 2. If you've run `simd` before, you may need to reset your database before starting a new
-    testnet: `$ ./simd unsafe-reset-all` 
-3. `$ ./simd init [moniker]`. This will initialize a new working directory, by default at 
-    `~/.simapp`. You need a provide a "moniker," but it doesn't matter what it is.
+    testnet. You can reset your database with the following command: `$ ./simd unsafe-reset-all`.
```

Comment:
```suggestion
    testnet. You can reset your database with the following command: `$ ./build/simd unsafe-reset-all`.
```

locally for me the binary gets stored in ./build, is this the same for you?

Reply:
It does for me too, which is why the first step states the following:

> The following instructions are run from inside the `build` directory.

Maybe this could be more clear?

Reply:
oh whoops, missed that part.